lib/SILOptimizer/FunctionSignatureTransforms/ExistentialSpecializer.cpp

```diff
@@ -158,6 +208,14 @@ findIfCalleeUsesArgInDestroyUse(SILValue Arg,
   }
 }
 
+/// Helper function to ensure that the argument is not InOut or InOut_Aliasable
```

Comment:
Added to filter out inout args, which otherwise would be fed to openExistentialRef as non-objects!